spec/gameSpec.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+describe('Game', function() {
+  let game
+
+  beforeEach(function() {
+    game = new Game();
+    game.beginFrame();
+  })
+
+  it('begins a new frame',function() {
+    expect(game.turn).toEqual(1)
+  })
+
+  it('has an empty scorecard at first', function() {
+    expect(game.scorecard).toEqual([])
+  })
+
```

Comment:
These tests strike me as testing state / implementation. pun intended. 

I say this because it's the same feedback I got from Alice last weekend.

Is it essential that the scores are stored in array? rather than `expect(game.scorecard).toEqual([])` you could do something like: 
`expect(game.scorecard.length).toBe(0)`

instead of `expect(game.scorecard[0]).toEqual([ 7 ])` you could do:
`expect(game.totalPoints).toBe(7 )`

it's a subtle difference, but I think this is what coaches mean when they tell us to test behaviour over state. I'm still learning this concept myself though really.